packages/nuxi/src/commands/test.ts

```diff
@@ -0,0 +1,32 @@
+import consola from 'consola'
+import { loadKit } from '../utils/kit'
+import { defineNuxtCommand } from './index'
+
+export default defineNuxtCommand({
+  meta: {
+    name: 'test',
+    usage: 'npx nuxi test',
+    description: 'Run tests'
+  },
+  async invoke (argv) {
+    process.env.NODE_ENV = process.env.NODE_ENV || 'test'
+    const rootDir = process.cwd()
+
+    const { loadNuxtConfig } = await loadKit(rootDir)
+    const config = await loadNuxtConfig({ rootDir })
+
+    // TODO: support Jest and other runners
+    consola.info('Starting Vitest...')
+
+    // TODO: prompt auto install if test-utils is not installed
+    let start: typeof import('@nuxt/test-utils/vitest')['default'] | undefined
+    try {
+      start = await import('@nuxt/test-utils/vitest').then(m => m.default)
```

Comment:
(we could use named `startVitest` export to avoid this)
```suggestion
      start = await import('@nuxt/test-utils/vitest').then(m => m.default || m)
```